_2017_pages/cfp.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: 2017_default
+permalink: /cfp
+title: Call for proposals is now open
+---
+    <!--background color-->
+
+    <section id="conference" class="cfp cfp-container cfp-container-top">
+      <h1>{{ page.title }}</h1>
+      <div class="normal-text">
+         <a href ="#" class="btn btn-sm normal-text">Click here to submit</a>
+      </div>
+    
+      <h2 class="first">Key Dates</h2>
+      <p class = "normal-text">Deadline for CFP: TBD</p>
+      <p class = "normal-text">Date of final submission: TBD</p>
+
+      <h2 class="first">What we are looking for</h2>
+      <p class="normal-text">
+      Sometimes in our lives we all have pain<br>
+      We all have sorrow<br>
+      But if we are wise<br>
+      We know that there's always tomorrow<br>
+      Lean on me, when you're not strong<br>
+      And I'll be your friend<br>
+      I'll help you carry on<br>
+      For it won't be long<br>
+      'Til I'm gonna need<br>
+      Somebody to lean on<br>
+      </p>
+
+      <h2 class="first">Format of conference</h2>
+      <p class=normal-text>Insert overview of how: (a) self recording (b) discussion will work</p>
+      <h3 class="first">Self-recording</h3>
+      <p class=normal-text>Put on a shirt. Pants optional.</p>
+      <h3 class="first">Discussion</h3>
+      <p class=normal-text>Put on a different shirt. Pants still optional.</p>
+
+      <h2 class="first">Code of conduct</h2>
+      <p class=normal-text>We have one! -- INSERT DETAILS HERE --/p>
```

Comment:
We should have a dedicated Code of Conduct page. Might ask for your help on this after CFP page is up

Reply:
Pinged the diversity committee folks to get help on this so it's not all on you :) Thank you for the reminder

Reply:
Sid I hope you don't mind I added the link 

![image](https://user-images.githubusercontent.com/7213793/89105760-e49b4880-d3d8-11ea-87f5-9ec24fc5abd3.png)